src/pegout/components/PegoutOption.vue

```diff
@@ -97,7 +105,7 @@
     <v-card-item class="flex-grow-1 d-flex flex-column align-left justify-end">
         <v-btn
           v-if="!formState.matches(['loading'])"
-          :disabled="!isReadyToCreate || formState.matches(['goingHome'])"
+          :disabled="!isReadyToCreate || formState.matches(['goingHome']) || !isValidBtcAddress"
```

Comment:
Default option's button is always disabled.
```suggestion
          :disabled="!isReadyToCreate || formState.matches(['goingHome']) || (isFlyover && !isValidBtcAddress)"
```